build.zig

```diff
@@ -562,6 +562,10 @@ fn addCmakeCfgOptionsToExe(
                 else => |e| return e,
             };
             exe.linkSystemLibrary("unwind");
+            if (std.c.versionCheck(.{ .major = 2, .minor = 28 }).ok) {
+                // see https://github.com/bminor/glibc/commit/1a2f44a848663036c8a14671fe0faa3fed0b2a25, included in glibc 2.28 and newer
```

Comment:
I'd recommend linking the official repository and not a random mirror on GitHub:
https://sourceware.org/git/?p=glibc.git;a=commit;h=1a2f44a848663036c8a14671fe0faa3fed0b2a25

Reply:
You're right.